README.md

```diff
@@ -27,11 +27,64 @@ security of the system, such as auditing and explainability features.
 
 [SIG-Security charter](governance/charter.md) outlines the scope  of our group activities, as part of our [governance process](governance) which details how we work.
 
-## Members
+## Communications
+
+Anyone is welcome to join our open discussions of SIG-Security projects and share news related to the group's mission and charter. Much of the work of the group happens outside of SIG-Security meetings and we encourage project teams to share progress updates or post questions in these channels:
+
+* [Email list](https://lists.cncf.io/g/cncf-sig-security)
+* [CNCF Slack](https://slack.cncf.io/) #sig-security channel
+
+## Meeting Time
+
+The Security SIG group meets every Wednesday at 10:00am PT (USA Pacific):
+
+Meeting minutes and agenda: https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g/
+Join: https://zoom.us/my/cncfsigsecurity
+
+One tap mobile:
+* +16465588656,,7375677271# US (New York)
+* +16699006833,,7375677271# US (San Jose)
+
+Dial by your location:
+* +1 646 558 8656 US (New York)
+* +1 669 900 6833 US (San Jose)
+* 877 369 0926 US Toll-free
+* 855 880 1246 US Toll-free
+Meeting ID: 737 567 7271
+Find your local number: https://zoom.us/u/alwlmxlNn
+
+[List of meetings](#meeting-list)
+
+## In Person Meetings
+
+Please let us know if you are going and if you are interested in attending (or helping to organize!) an in-person meetup. Create a [github issue](https://github.com/cncf/sig-security/issues/new) for an event and add to list below:
+
+* KubeCon + CloudNativeCon, San Diego, CA - Nov 18 – 21, 2019 - [issue#128]
+
+[Past Events](past-events.md)
+
+## New Members
+
+If you are new to the group, check out our [New Members Page](NEWMEMBERS.md)
+
+## Related Groups
+
+* [Kubernetes Policy Working Group](https://github.com/kubernetes/community/tree/master/wg-policy)
+* [Kubernetes SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth)
+* [NIST Big Data WG](https://bigdatawg.nist.gov/)
+
+## History
+
+* SAFE WG - renamed to CNCF Security SIG
+* [(Proposed) CNCF Policy Working Group](/policy-wg-merging.md) - Merged into SAFE WG
+
+## Members list
 
```

Comment:
it might be worth having a `## SIG Chairs` header here and a `## Additional SIG members` (or something) header in the collapsed section

Reply:
Agreed! Let me add a subheader here.

Reply:
non-blocking: Would be great to link to governance definition of membership here.  (It's awkward not to have some kind of intro, but this is SO much better and would be completely fine to submit as incremental change after this is merged)

Reply:
do you mean like the [new members](https://github.com/cncf/sig-security/blob/master/NEWMEMBERS.md) page or something else

Reply:
I suppose its https://github.com/cncf/sig-security/blob/master/governance/roles.md#role-of-members, but i think we can link both, would be useful.